search-processors/src/main/java/org/opensearch/searchpipelines/questionanswering/generative/GenerativeQAResponseProcessor.java

```diff
@@ -171,7 +171,7 @@ public SearchResponse processResponse(SearchRequest request, SearchResponse resp
                         PromptUtil.getPromptTemplate(systemPrompt, userInstructions),
                         answer,
                         GenerativeQAProcessorConstants.RESPONSE_PROCESSOR_TYPE,
-                        jsonArrayToString(searchResults)
+                        Collections.singletonMap("metadata", jsonArrayToString(searchResults))
```

Comment:
can use the key `"search_results"`?

Reply:
Ok let's make it consistent with the previous tests.